### PR TITLE
Add config and build files to run Windows tests on Kokoro

### PR DESCRIPTION
**Pull request by matt-kwong**
This adds the necessary files to run the current Jenkins Windows tests on Kokoro.

### CODE REVIEW

File: buildscripts/kokoro/windows.cfg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Config file for internal CI
+
+# Location of the continuous shell script in repository.
+build_file: "grpc-java/buildscripts/kokoro/windows.bat"
+timeout_mins: 360
```
Comment (ejona86) [inline buildscripts/kokoro/windows.cfg]: That's 6 hours? How about we limit it to something smaller, say 45 minutes?

File: buildscripts/kokoro/windows.bat [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+cd /d %~dp0\..\..
+
+set WORKSPACE=T:\src\github\grpc-java
+set ESCWORKSPACE=%WORKSPACE:\=\\%
+
+mkdir grpc-java-helper
+
+@rem Install 7za
+wget http://www.7-zip.org/a/7za920.zip
+unzip 7za920.zip
```
Comment (ejona86) [inline buildscripts/kokoro/windows.bat]: If we have unzip then we don't need 7za. Our other build machines don't have unzip. But it'd be annoying to support both in make_dependencies.bat. Add a TODO here to swap make_dependencies.bat to use unzip instead.

Comment (matt-kwong) [inline buildscripts/kokoro/windows.bat]: Is Jenkins the only place `make_dependencies.bat` is used currently?

Comment (ejona86) [inline buildscripts/kokoro/windows.bat]: Yeah, although it is a nice feature that `make_dependencies.bat` can be run manually. But depending on unzip vs 7za doesn't really make it any harder to run manually.

**Pull request merged**